docs/spec.md

```diff
@@ -101,14 +102,34 @@ Each JSON message must be an object containing a field called `op` which identif
 - `op`: string `"status"`
 - `level`: 0 (info), 1 (warning), 2 (error)
 - `message`: string
+- `id`: string | undefined. Optional unique identifier that can be used for clearing the status.
 
 #### Example
 
 ```json
 {
   "op": "status",
   "level": 0,
-  "message": "Some info"
+  "message": "Some info",
+  "id": "someId"
+}
+```
+
+### Clear Status
+
+- Informs the client that a previously sent status message shall be cleared.
+
+#### Fields
+
+- `op`: string `"clearStatus"`
+- `id`: string, the id of the status message to be cleared.
```

Comment:
@snosenzo's [comment](https://github.com/foxglove/ws-protocol/pull/783?notification_referrer_id=NT_kwDOAI0la7MxMTU0NTY5MzU2MTo5MjUwMTU1&notifications_query=is%3Aunread#pullrequestreview-2183977854) got me thinking that this could also be a string array to allow clearing of multiple status at once

```suggestion
- `id`: string[], ids of the status messages to be cleared.
```

Reply:
yeah I think that's a good improvement